pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "builddsl"
-version = "1.0.0"
+version = "1.0.1"
```

The bump happens when I run `slap release` :)

```suggestion
version = "1.0.0"
```

---

ups, sorry - I did `slap release` on my fork repo =D ...I'll revert it now